py4web/utils/grid.py

```diff
@@ -907,7 +907,7 @@ def _make_field_header(self, field, field_index, sort_order):
             if "label" in field.__dict__
             else field.name
         )
-        heading = title(heading)
+        heading = XML(title(heading))
```

Comment:
why this? seems unsafe. titile can return XML(...) if needs to

Reply:
I don't have a good answer. I have a grid column label that is "Load<br />Order" and it display just that if I don't have XML around it. I also tried to pass the heading as XML("Load<br />Order") but that didn't work either.

But, if it isn't working maybe it should be fixed in title instead of in grid. I can do some more testing tomorrow.

Reply:
I looked at this a bit more and, unless I'm missing something, I don't see that title converts HTML tags.

Reply:
I just pushed a change that may address the issue in another way. Could you please take a look? https://github.com/web2py/py4web/pull/636